DuckDuckGo/Subscription/ViewModel/SubscriptionFlowViewModel.swift

```diff
@@ -249,41 +250,23 @@ final class SubscriptionFlowViewModel: ObservableObject {
     private func backButtonEnabled(_ enabled: Bool) {
         state.canNavigateBack = enabled
     }
+
+    // MARK: -
     
-    @MainActor
-    func onAppear() async {
+    func onFirstAppear() async {
         DispatchQueue.main.async {
             self.resetState()
-            self.webViewModel.navigationCoordinator.navigateTo(url: self.purchaseURL )
         }
-        Pixel.fire(pixel: .privacyProOfferScreenImpression, debounce: 2)
```

Comment:
side question, are you going to remove the debounce feature from Pixel?

Reply:
We can remove from these pixels later, but no plans to remove from the class.  It may become useful later.